packages/flutter/lib/src/material/drawer.dart

```diff
@@ -100,13 +103,29 @@ class Drawer extends StatelessWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// An (optional) name for the drawer used by accessibility frameworks.
```

Comment:
Not just accessibility frameworks, anything that uses semantics. For example, a search engine could inspect an app's contents using the semantics API. That's why it's not an accessibilty API, but a semantics API.